lib/attentive_sidekiq/middleware/server/attentionist.rb

```diff
@@ -3,10 +3,16 @@ module Middleware
     module Server
       class Attentionist
         def call(worker_instance, item, queue)
-          Suspicious.add(item)
+          reliable_job = item["adblock_reliable"]
```

Comment:
Fine for our use can, but if we ever plan to submit this as a PR to the original gem we should use a more generic name